cmd/create.go

```diff
@@ -44,21 +53,21 @@ func createCmdFunc(cmd *cobra.Command, args []string) {
 	switch createType {
 	case "manager":
 		fmt.Println("create manager called")
-		err := create.NewTritonManager(remoteBackend)
+		err := create.NewTritonManager(remoteBackend, silentMode)
```

Comment:
If you use Viper I don't think you'll need to pass `silentMode` around. Since Viper sets itself up as a global var